.github/workflows/publish-website.yml

```
@@ -46,6 +46,15 @@ jobs:
           deriv_me_site_id: ${{ secrets.DERIV_ME_SITE_ID }}
           deriv_be_site_id: ${{ secrets.DERIV_BE_SITE_ID }}
           webflow_general_site_id: ${{ secrets.WEBFLOW_GENERAL_SITE_ID }}
+      
+      - name: Trigger Scraper workflow
+        id: trigger-scraper
+        uses: peter-evans/repository-dispatch@v3
```

```suggestion
        uses: peter-evans/repository-dispatch@v3
```
I think we can use github native functionalities for this instead of a third party action.
Please check possibility of it.